README.md

```diff
@@ -103,33 +114,62 @@ easy_install --upgrade "ibmcloudant>=0.0.27"
 
 ### Authentication
```

Comment:
Now I see the `Getting started` section a little bit lengthy and lost its point a little bit. 
I am thinking about moving out the full `Authentication` section from `Getting started`.
Maybe even remove `Getting started` section naming. And use `Code examples` as H2.

Additionally, the sentence: 
```
The code examples below will follow the
[authentication with environment variables](#authenticate-with-environment-variables).
```
Would be good to move into the `Code examples` section.
After this change the TOC build up would be more structured for a user.

Reply:
Yes, it is a good idea! I fixed it in 576a9cb.